README.md

```diff
@@ -72,33 +82,42 @@ docker run -p 5090:5090 -d oss_report:$(cat VERSION)
 2. Generate a [SQLite](https://docs.python.org/2/library/sqlite3.html) database file in the `analyze/` folder.
 
 ```
-cat analyze/schema.sql | sqlite3 analyze/thing.db
+cat analyze/schema.sql | sqlite3 thing.db
 ```
 
-3. Populate a CSV called `thing.csv` with your users.
+3. Populate a CSV with your users.
 
-Headers:
+* `user_name`: Github user name (e.g. `jameslamb`)
+* `full_name`: full first and last name (`James Lamb`)
+
+For example:
 
 ```
-user_name,full_name
+echo "user_name,full_name" > thing.csv
+echo "jameslamb,James Lamb" >> thing.csv
+echo "bburns632,Brian Burns" >> thing.csv
+echo "jayqi,Jay Qi" >> thing.csv
```

Comment:
@jayqi @bburns632 hope it's ok I use you in my docs 😀 

Here's the result of this over the last 30 days (if you're curious):

```
user_name   total_contributions  num_unique_repos
----------  -------------------  ----------------
bburns632   36                   4
jameslamb   51                   10
jayqi       89                   9
```